node/transaction.go

```diff
@@ -113,6 +130,8 @@ func (localNode *LocalNode) startRequestingSigChainTxn() {
 					continue
 				}
 
+				requestedHashCache.Set(info.hash, struct{}{})
```

Comment:
Node still would tried to request SigChain.Hash again and again if `localNode.requestSignatureChainTransaction(nbr, hash)` return err.